packages/plugin-core/src/features/TipOfTheDayWebview.ts

```diff
@@ -0,0 +1,198 @@
+import { VSCodeEvents } from "@dendronhq/common-all";
+import { MetadataService } from "@dendronhq/engine-server";
+import _md from "markdown-it";
+import * as vscode from "vscode";
+import {
+  DisplayLocation,
+  FeatureShowcaseUserResponse,
+  IFeatureShowcaseMessage,
+} from "../showcase/IFeatureShowcaseMessage";
+import { AnalyticsUtils } from "../utils/analytics";
+
+/**
+ * Side Panel webview that shows the tip of the day
+ * TODO: Add functionality
+ *  - let user rotate tips
+ */
+export default class TipOfTheDayWebview implements vscode.WebviewViewProvider {
+  private _webview: vscode.WebviewView | undefined;
+  private _tips: IFeatureShowcaseMessage[];
+  private _curTipIndex;
+
+  private BUTTON_CLICKED_MSG = "buttonClicked";
+  private TIP_SHOWN_MSG = "loaded";
+
+  private get _currentTip(): IFeatureShowcaseMessage {
+    return this._tips[this._curTipIndex];
```

Comment:
Tip of the day resulted in an error for me: `An error occurred while loading view: dendron.tip-of-the-day`. 
An undefined tip is returned from here with this._curTipIndex being floating-point number.

Reply:
🤦🏼 - good catch!